### PR TITLE
feat: add workflow for new contributors

### PR DESCRIPTION
**Pull request by raisedadead**
Sometimes we miss congratulating new contributors because sometimes we merge PRs automatically when requirements are met. This should ensure we are not missing the courtesy of welcoming new contributors.

Also. I updated some naming conventions for our actions earlier. This PR updates that slightly too.

### CODE REVIEW

**Review by Sboonny — APPROVED** [review]
LGTM 🎉

**Review by majestic-owl448 — APPROVED** [review]

**Comment by ieahleen** [comment]
awesome!

File: .github/workflows/welcome-new-lgtm.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: wow-actions/welcome@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FIRST_PR_MERGED: |
+            #### :sparkles: :tada: **AWESOME!** :tada: :sparkles:
+
+            Hi @{{ author }},
```
Comment (ShaunSHamilton) [inline .github/workflows/welcome-new-lgtm.yml:15]: I am just waiting for the day this fails with:

> Hi undefined,

Comment (raisedadead) [reply .github/workflows/welcome-new-lgtm.yml:15]: Will it? I am not sure how to accurately test this. I used the template they gave us.

Comment (Sboonny) [reply .github/workflows/welcome-new-lgtm.yml:15]: 🤣🤣

Comment (raisedadead) [reply .github/workflows/welcome-new-lgtm.yml:15]: Ah - I see what you did here Shaun.

Comment (ShaunSHamilton) [reply .github/workflows/welcome-new-lgtm.yml:15]: > Will it?

Only if any of this API is written in JS. Otherwise, it is just a wish of mine.

Comment (raisedadead) [reply .github/workflows/welcome-new-lgtm.yml:15]: Goodbye Shaun. 😡

**Pull request merged**